budcamp/Homework3/Homework3/Poland.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class Poland: History, ICountry
```

Comment:
Same here, for a relationship between types `Poland` and `History` composition makes much more sense then inheritance.